docs/stats.md

```diff
@@ -0,0 +1,38 @@
+# Stats
+
+## Overview
+<!-- TODO: move module definitions here? -->
+A set of methods used to calculate statistics over the course of an
+experiment.
+Each such method is a higher-order function returning a `track_stat` method
+that takes in an Experiment object, an action key, and the address of the
+Keeper that executed said action.
+The higher-order function can take in whatever parameters it may need to
+provide to `track_stat`.
+Each `track_stat` function will be run after a Keeper executes in an action
+in a timestep, so an intended design pattern is to guard your logic in
+`track_stat` w/ a predicate that ensures you're only executing the method
+in response to a specific action you were listening for.
+
+## Common Conventions
+
+## Functions
+### ilk_price
+
+Updates the ilk values in `stats`.
```

Comment:
I think for each of these, we should also mention what parameters the HOF takes, and explain what condition / action key they're "listening" for.
e.g. for `ilk_price`, it takes in `ilk_id`, and only updates `state["stats"]["ilk_price"][ilk_id]` when the action passed in had a key `POKE`

Reply:
and also, for each, describe the shape / type of the stat being saved.
e.g. for `ilk_price`, it writes a dict that maps from Ilk ids to Ilk spot values (converted to floats)